swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
+from operator import attrgetter
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None):
+        if inventory == None:
+            self.inventory = []
+        else:
+            self.inventory = inventory 
```

Comment:
Good job avoiding a mutable default value! One small nitpick, use `is None` when checking whether an object is None.